themes/harper/layouts/shortcodes/lang.html

```
@@ -12,10 +12,8 @@
     <span class="languages-prefix">
         {{ i18n "also-available-in" | default "Also available in" }}: </span>
     {{ range $i, $lang := $languages }}
-      <a href="{{ $lang.url }}" hreflang="{{ $lang.lang }}" class="language-link"
-         aria-label="{{ i18n "read-in-language" | default "Read this post in %s" | printf $lang.name }}"
-         data-tinylytics-event="navigation.language"
-         data-tinylytics-event-value="{{ $lang.lang }}">
+      <a href="{{ $lang.url }}" hreflang="{{ $lang.lang }}" class="language-link" 
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)